README.md

```diff
@@ -1 +1,14 @@
-# hello-microverse-project
\ No newline at end of file
+# hello-microverse-project
+Hello Microverse project
+
+Built With HTML (Hypertext Markup language) and CSS (Cascading Style Sheet)
+
```

Comment:
- [ ] [ OPTIONAL ] Your README looks great 👍🏻  but I think you can improve it by making it more professional. A professional README file should have a project title, project description, list of technology used in the project, instructions to set the project up locally, and a screenshot of your project. please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.